File: pyosx/v5-binding/Say.m
#import <AVFoundation/AVFoundation.h>

void say(char* text) {
    NSString* string = [NSString stringWithFormat:@"%s", text];
    AVSpeechSynthesizer *synthesizer = [[AVSpeechSynthesizer alloc] init];  // init speaker
    AVSpeechUtterance *utterance = [[AVSpeechUtterance alloc] initWithString:string];  // init utterance
    [synthesizer speakUtterance:utterance];  // speak utterance
    CFRunLoopRunInMode(kCFRunLoopDefaultMode, 3.0, false);  // allow time for speaking to finish
}

int main(int argc, const char * argv[]) {
    @autoreleasepool {
        say("I wish oranges came in purple.");
    }
    return 0;
}

File: pyosx/v0-pyobjc/say.py
from AVFoundation import AVSpeechUtterance, AVSpeechSynthesizer
import time


def say(text):
    synth = AVSpeechSynthesizer.alloc().init()  # init speaker
    utterance = AVSpeechUtterance.speechUtteranceWithString_(text)  # init utterance
    synth.speakUtterance_(utterance)  # speak utterance
    time.sleep(3)  # allow time for speaking to finish


if __name__ == '__main__':
    say("I wish oranges came in purple.")